conan_patches/onnxruntime/conandata.yml

```diff
@@ -0,0 +1,13 @@
+patches:
+  "1.18.1":
+    - patch_file: "patches/android-64.diff"
+      patch_description: |
+        Fixes error for arm64-v8a targets:
+        onnxruntime/core/mlas/lib/mlasi.h:366:11: error: unknown type name 'bfloat16_t'
```

Comment:
and what's the reason? Is it about minimum API level?

Reply:
I can't answer this question. 
`bfloat16` support seems introduced in [Armv8.6-A](https://developer.arm.com/documentation/109697/2025_09/Feature-descriptions/The-Armv8-6-architecture-extension) in 2019, but onnxruntime/1.16.1 contains [flags for armv8.2](https://github.com/microsoft/onnxruntime/blob/v1.18.1/cmake/onnxruntime_mlas.cmake#L371). This might cause issues on older chips where these optimized instructions are unavailable. 

However, we should be safe. The MMAI models are not using bfloat16 arithmetic. I have tested the model with Samsung Galaxy A71 which has an ARM Cortex-A76 and Cortex-A55, both [implementing Arm8.2-A](https://en.wikipedia.org/wiki/ARM_Cortex-A76) and there have been no issues.